test/docker-compose-selenium-test.yml

```diff
@@ -9,7 +9,7 @@ services:
     depends_on:
       - wikibase
     volumes:
-      - ./log/selenium/:/usr/src/app/log/
+      - ../test/log:/usr/src/app/log
```

Comment:
This is a bit of a functional hack for the case of `test-example` but still works same as before in all other cases.